src/coreclr/debug/daccess/request_common.h

```diff
@@ -113,6 +113,11 @@ inline bool IsBackgroundGCEnabled()
     return (g_gcDacGlobals->minor_version_number & 2) == 0;
 }
 
+inline bool IsDatasEnabled()
+{
+    return (g_gcDacGlobals->minor_version_number & 4) == 0;
```

Comment:
should `4` be defined as a constant somewhere? Same for other constants above actually.

Reply:
It is a questionable practice to overload version number to mean something else. Should this rather be a new variable?